wp-content/plugins/wporg-learn/inc/redirects.php

```diff
@@ -52,14 +56,17 @@ function wporg_learn_redirect_old_urls() {
 	foreach ( $redirects as $source => $destination ) {
 		if ( str_starts_with( $request, $source ) ) {
 			$redirect = $destination;
+			$code = 301;
 
 			// Append any extra request parameters.
 			if ( strlen( $request ) > strlen( $source ) ) {
 				$redirect .= substr( $request, strlen( $source ) );
 			}
 
-			wp_safe_redirect( $redirect );
-			die();
+			str_starts_with( $destination, 'https://' )
+				? wp_redirect( $redirect, $code, 'Learn WordPress' )
+				: wp_safe_redirect( $redirect, $code );
+			exit;
```

Comment:
The alternative here is to add `wordpress.tv` to the 'allowed_redirect_hosts' filter, which is probably the route I'd go. That way it passes through `wp_safe_redirect()` as it's then a safe-destination.

Reply:
Ah great, that's exactly the input I was after, cheers